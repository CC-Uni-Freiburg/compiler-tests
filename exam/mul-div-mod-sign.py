#in=17
#in=4
#golden=4010-40-10-401040-10
x = input_int()
y = input_int()
print(x // y)
print(0)
print(x % y)
print(0)
print(-x // y)
print(0)
print(-x % y)
print(0)
print(x // -y)
print(0)
print(x % -y)
print(0)
print(-x // -y)
print(0)
print(-x % -y)
print(0)
