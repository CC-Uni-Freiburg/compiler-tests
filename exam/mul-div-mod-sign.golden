4010-40-10-401040-10
